### PR TITLE
Changes to Win start menu group shortcut 

### PR DESCRIPTION
**Pull request by rgozim**
Fixes logic error, allowing for configuration of windows or mac options regardless of OS.

### CODE REVIEW

**Comment by jburel** [comment]
Changes will be required for insight to work. 5.5.1-SNAPSHOT cannot be used "out of the box"

```
openmicroscopy/PackagerPlugin.groovy: 70: [Static type checking] - Cannot set read-only property: outputTypes
 @ line 70, column 17.
                   importer.outputTypes = outputTypes

```

After removing that line, I was able to run ``packageApplicationDmg`` and ``packageImporterApplicationDmg``
It was built much quicker that before
I will test on Windows on monday

**Comment by rgozim** [comment]
The work is in place for updating insight to use the latest plugin. A PR will be open as soon as Insight as I've also tested on Windows.

See:
https://github.com/rgozim/omero-insight/commit/ed0e27f6e772eb9a62be7266980d366f4d1f4ce0

**Comment by jburel** [comment]
Tested the creation of Exe on windows. This works as expected

File: README.md [hunk 1]
```diff
@@ -29,3 +29,38 @@ exist for the various components. For example, to create an OMERO.importer .dmg
...
+    }
+}
+```
+
+Full custom example:
+
+```gradle
+deploy {
+    someApp {
+        outputTypes = ["msi", "dmg"]                            // this deploy config will create tasks for msi or dmg installers, depending on OS
```
Comment (jburel) [inline README.md]: use ``exe`` since it is was what we are deploying

**Comment by jburel** [comment]
I combine this PR and the insight "develop PR" in travis 
Travis file has been modified to use ``omero-javapackager-plugin`` snapshot
https://travis-ci.org/jburel/omero-insight/jobs/540683182
the job failed during the linux build phase

```
An exception occurred applying plugin request [id: 'org.openmicroscopy.packager']
> Failed to apply plugin [class 'org.openmicroscopy.JavaPackagerPlugin']
   > Could not create an instance of type org.openmicroscopy.extensions.implementation.DefaultInstallOptions.
      > java.lang.NullPointerException (no error message)
```

**Comment by rgozim** [comment]
@jburel Latest commit solves the reported error. Issue was with Gradle 5.2.

**Comment by jburel** [comment]
```
* What went wrong:
A problem occurred configuring root project 'omero-insight'.
> Could not create task ':packageApplicationDmg'.
   > Extension with name 'dmg' does not exist. Currently registered extension names: [ext, exe, msi]
```

**Comment by jburel** [comment]
https://travis-ci.org/jburel/omero-insight/builds/540683178 is back to green.
I will do a test on Windows.

**Comment by jburel** [comment]
Creation of exe works on windows
Merging and tagging to 5.5.1

**Pull request merged**